src/main/java/it/grid/storm/tape/recalltable/resources/TasksResource.java

```diff
@@ -20,17 +20,22 @@
  */
 package it.grid.storm.tape.recalltable.resources;
 
+import com.google.common.collect.Lists;
+
 import it.grid.storm.config.Configuration;
 import it.grid.storm.persistence.model.TapeRecallTO;
 import it.grid.storm.tape.recalltable.TapeRecallCatalog;
 import it.grid.storm.tape.recalltable.TapeRecallException;
 import it.grid.storm.tape.recalltable.model.TapeRecallStatus;
+import jersey.repackaged.com.google.common.collect.Maps;

```

Comment:
Where is this Maps used? Is it correct to use the jersey repackaged version instead of the Guava one?

Reply:
cmd-shift-o command wrongly added this instead of the right one